### ISSUE TITLE
Limitation of number of private app for PRO plan

### ISSUE THREAD

**Issue by supindepa** [thunkable/thunkable-issues#1186]
**Is your feature request related to a problem? Please describe.**
I am currently subscribed to the pro plan.
There is a limit of 50 PRIVATE apps.
In thunkable, the delete and duplicate buttons are lined up so that I can accidentally delete an app.
Even in that case, there is currently no way for me to recover them.
Therefore, it has become necessary to always have an original and a copy of each live app.
This is an unfortunate situation that prevents me from producing 50 apps.

**Describe the solution you'd like**
Therefore, I would like to propose the following solution to your company.
1. Create a recycle bin function for deleted apps (i.e., make it possible to recover the deleted apps).
2. Set the number of private apps in the professional plan to about 100.

**Which platform are you asking for?**
Thunkable ✕ (x.thunkable.com)

**Additional context**
I think some developers have similar problems.
Please consider this at your company.
We look forward to your official response.

**Comment by tatiang** [comment]
Yes, this is a significant problem with the pricing structure of Thunkable. Please see this topic I created:

https://community.thunkable.com/t/please-consider-increasing-private-project-limit-for-pro-users/2075025
